src/data/guilty_gear_xrd_rev_2/jam.yml

```diff
@@ -0,0 +1,80 @@
+---
+name: Jam Kuradoberi
+moves:
+  normals:
+    - 5P
+    - 5K
+    - c.S
+    - f.S
+    - 5H
+    - 5D
+    - 6P
+    - 6K
+    - 6H
+    - 2P
+    - 2K
+    - 2S
+    - 2H
+    - 2D
+    - j.P
+    - j.K
+    - j.S
+    - j.H
+    - j.D
+  specials:
+    - name: Hochifu
+      inputs: 546
+      follow_ups:
+        - name: Zekkei
+          inputs: P
+          follow_ups:
+            - name: Goushao
+              inputs: P
+        - name: Dowanga
```

Comment:
Dowanga is a follow-up to Zekkei, not Hochifu.

Reply:
whoops!

Reply:
updated